src/afl-tmin.c

```diff
@@ -37,6 +37,12 @@
 #include "forkserver.h"
 #include "sharedmem.h"
 #include "common.h"
+#include "afl-fuzz.h"
+#include "list.h"
+#ifdef USE_PYTHON
+#include "afl-fuzz-python.c"
```

Comment:
that is not how you do it - you add both files to the compile line in GNUmakefile. no need of the ifdef either. just look how afl-showmap is doing it.